elastic/kibana
# [Embeddables] Fix missing panel references when Duplicate, Copy to Dashboard, or Save as new dashboard

Fixes #183551

## Summary

Fixes missing references on React embeddables when Duplicate, Copy to Dashboard, or Save as new dashboard.


To test this:
1. Start Kibana with `yarn start --run-examples`.
2. Install at least two sample datasets.
3. Create a new dashboard and add a Field list panel.
4. Change the data view in the Field list panel to something other than the default data view.
5. Duplicate the panel and ensure the data view does not revert to the default.
6. Copy the panel to a new or existing dashboard and ensure the data view does not revert.
7. Save the dashboard to a new dashboard and ensure the data view does not revert.
